src/php/Runtime/RuntimeFactory.php

```diff
@@ -5,30 +5,15 @@
 namespace Phel\Runtime;
 
 use Gacela\Framework\AbstractFactory;
-use Phel\Runtime\Exceptions\PhelRuntimeException;
 
-/**
- * @method RuntimeConfig getConfig()
- */
 final class RuntimeFactory extends AbstractFactory
 {
-    /**
-     * @throws PhelRuntimeException
-     */
     public function getRuntime(): RuntimeInterface
     {
         if (RuntimeSingleton::isInitialized()) {
             return RuntimeSingleton::getInstance();
         }
 
-        $runtimePath = $this->getConfig()->getApplicationRootDir()
-            . DIRECTORY_SEPARATOR . 'vendor'
-            . DIRECTORY_SEPARATOR . 'PhelRuntime.php';
-
-        if (!file_exists($runtimePath)) {
-            throw PhelRuntimeException::couldNotBeLoadedFrom($runtimePath);
-        }
-
-        return require $runtimePath;
+        return RuntimeSingleton::initialize();
```

Comment:
I am not quite sure this is what we want... 🤔 because this way you would ignore the configuration that knows which paths and namespaces your program is aware of. Previously that config was under composer extra fields but now is in your `phel-config.php` file at the root level.

Previously the composer plugin was generating this file (when you did an install or update), but we deprecated that thing and now we generate the PhelRuntime manually when the user wants (see [PR](https://github.com/phel-lang/phel-lang/pull/346)). This is your new friend: `phel runtime`